### PR TITLE
Group notes by book

### LINKED ISSUE #2: Change the view in <username>/notes to group notes by books
When you create multiple notes for a book we want to group those notes under the same book. 

E.g. If you [check my notes](https://booknshelf.com/@tigran/notes) you will see that I have two notes for "Thinking fast and slow" book but we display these notes separately. I want us to group these notes by a book so we will have 2 notes and 1 book.

### PR DESCRIPTION
**Pull request by ShikherVerma**
Fixes #2

### CODE REVIEW

File: resources/views/notes.blade.php [hunk 1]
```diff
@@ -11,16 +11,20 @@
                     </p>
                 </div>
             @endif
-            @foreach ($notes as $note)
+            @php
+            $notesByBook = (collect($notes) ->groupBy("book_id") ->toArray())
```
Comment (tiggreen) [inline resources/views/notes.blade.php]: I'd suggest to have as less logic in the view as possible. Maybe we could move this part to [UserController](https://github.com/tiggreen/booknshelf.com/blob/master/app/Http/Controllers/UserController.php#L56)?

**Comment by tiggreen** [comment]
Hey @ShikherVerma this is great! Congrats on your first PR! Amazing work 🎉 I've a minor comment once that's addressed we can merge it!

**Comment by ShikherVerma** [comment]
Moved php code to `UserController`

**Comment by tiggreen** [comment]
Great work on this!

**Pull request merged**